packages/test-helpers/src/utils/numbers.js

```diff
@@ -1,30 +1,23 @@
 const { BN } = require('web3-utils')
 
 const bn = x => new BN(x)
+const pct = x => bigExp(x, 16)
 const bigExp = (x, y) => bn(x).mul(bn(10).pow(bn(y)))
-const maxUint = e =>
-  bn(2)
-    .pow(bn(e))
-    .sub(bn(1))
+const maxUint = e => bn(2).pow(bn(e)).sub(bn(1))
+const isBigNumber = x => x instanceof BN || (x && x.constructor && x.constructor.name === BN.name)
```

Comment:
Why not just use [`isBN()`](https://web3js.readthedocs.io/en/v1.2.9/web3-utils.html#isbn)?